packages/ui/src/walkthrough-card/WalkthroughCard.tsx

```diff
@@ -20,7 +20,7 @@ export interface WalkthroughCardProps extends ReactAriaLinkProps {
 export default function WalkthroughCard({
   data,
   walkthroughId,
-  "data-testid": testid = TESTID_WALTHROUGH_CARD,
+  "data-testid": testid = GET_TESTID_WALKTHROUGH_CARD(walkthroughId),
```

Comment:
Looking at this more now I think we want to refactor to this:

// get passed unique test id or default to nothing
`"data-testid": testid = "",`

// make testid unique if an override is passed, otherwise default to the walkthroughId
`data-testid={GET_TESTID_WALKTHROUGH_CARD(testid || walkthroughId)}`

As this is set up now, if a testid is passed it will lose the `walkthrough-card-` prefix. I would think we'd want all testids for the walkthrough cards to contain the prefix.

Reply:
Updated!